course_catalog/serializers.py

```diff
@@ -297,6 +297,11 @@ class CourseSerializer(BaseCourseSerializer, LearningResourceRunMixin):
     object_type = serializers.CharField(read_only=True, default="course")
     audience = serializers.ReadOnlyField()
     certification = serializers.ReadOnlyField()
+    department_slug = serializers.SerializerMethodField()
+
+    def get_department_slug(self, instance):
```

Comment:
You can just replace line 300 with `department_slug =  serializers.ReadOnlyField()` similar to audience and certification and remove `get_department_slug`

Reply:
Thank! Changed it.